ikarus/utils/eigendunetransformations.hh

```diff
@@ -58,4 +59,16 @@ namespace Ikarus {
     return {vec.data(), size};
   }
 
+  /** \brief Creates a Eigen::Vector from a given Dune::BlockVector. Each block should be a
+   * Dune::RealTuple of a given blockSize */
+  template <typename ScalarType, int blockSize>
+  Eigen::VectorX<ScalarType> convertDuneBlockToEigenVector(

```

Comment:
Can you think a bout a better name for this here. Can you also comment about that the result orders all dofs `FlatInterleaved`?

Look up how to write doxygen comments: 

```cpp

/**
 * \brief Adds two numbers.
 *
 * This function takes two numbers, adds them, and then returns the result.
 * 
 * \param x The first number to add.
 * \param y The second number to add.
 * \return The sum of the two numbers.
 */
int add(int x, int y) {
    return x + y;
}
```

Reply:
Apparently this function is not needed as there exists already `Dune::viewAsFlatEigenVector`. I removed this. See #223